README.md

```diff
@@ -106,6 +106,7 @@ Command Topics
 * [`box help`](docs/help.md) - Display help for the Box CLI
 * [`box legal-hold-policies`](docs/legal-hold-policies.md) - List legal hold policies
 * [`box metadata-cascade-policies`](docs/metadata-cascade-policies.md) - List the metadata cascade policies on a folder
+* [`box metadata-query`](docs/metadata-query.md) - Create a search using SQL-like syntax to return items that match specific metadata
```

Comment:
This probably will get overwritten by the automatic documentation generation which also creates `docs/*` files